hardhat.config.ts

```diff
@@ -51,6 +51,7 @@ export default {
 					apiKey: process.env.ETHERSCAN_API_KEY
 				}
 			},
+			gasMultiplier: 1.1,
```

Comment:
why?

Reply:
Because otherwise Goerli deploys tend to fail.